components/rating/package.json

```diff
@@ -19,12 +19,12 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.0",
-    "@spectrum-css/tokens": "^7.1.0"
+    "@spectrum-css/tokens": "^7.6.1"
   },
   "devDependencies": {
     "@spectrum-css/component-builder": "^4.0.3",
     "@spectrum-css/icon": "^3.0.35",
-    "@spectrum-css/vars": "^8.0.5",
+    "@spectrum-css/tokens": "^7.6.1",
```

Comment:
@venkateshjo I think this might have been part of our problem - we didn't add the tokens dependency here and _I think_ this might have been using the "old" colors via `vars`.